src/generators/static/tsConfigFileGenerator.ts

```diff
@@ -81,6 +81,9 @@ export function generateTsConfig(project: Project) {
     highLevelTsConfig.include.push("samples-dev/**/*.ts");
     highLevelTsConfig.compilerOptions["paths"] = {};
     highLevelTsConfig.compilerOptions["paths"][clientPackageName] = ["./src/index"];
+    restLevelTsConfig.include.push("samples-dev/**/*.ts");
```

Comment:
do we need to check for generateSamples?

Reply:
It's already under the condition of generateSample